web-ng/src/app/components/layer-dialog/layer-dialog.component.ts

```diff
@@ -68,6 +71,24 @@ export class LayerDialogComponent implements OnDestroy {
     );
   }
 
+  getFieldType() {
+    return {
+      icon: 'short_text',
+      label: 'Text',
+    };
+  }
+
+  initQuestion() {
```

Comment:
`init` implies it changes some internal state - here it just returns an empty group. Perhaps "createQuestionGroup" would be a better name?